src/GitLink/Extensions/PdbExtensions.cs

```diff
@@ -68,8 +68,15 @@ public static IEnumerable<string> FindMissingOrChangedSourceFiles(this PdbFile p
 
         private static bool AreEqualBuffers(byte[] first, byte[] second)
         {
-            Argument.IsNotNull(nameof(first), first);
-            Argument.IsNotNull(nameof(second), second);
+            if (first == null && second == null)
```

Comment:
> No need to get into a (possibly expensive, we don't know the implementation here) == operation just to check for null

I do occasionally use `object.ReferenceEquals` when the type I'm comparing does (or may) overload the `==` operator and I want to ensure high performance. But as almost no class does this, and when they do, they're generally supposed to be implemented as very fast, and [Premature Optimization is an anti-pattern](http://wiki.c2.com/?PrematureOptimization), I almost always prefer the more familiar `==` operator. 
In this case particularly, `byte[]` is an array, and arrays do not and cannot overload the `==` operator, so we're going to get the default reference equality check.

> And we could store the null result in a variable, even faster

the null result of what? The only storage we're reading from already come from local variables (the args) anyway.